app/views/ctc/questions/consent/edit.html.erb

```diff
@@ -0,0 +1,34 @@
+<% @main_question = t("views.ctc.questions.legal_consent.title") %>
+
+<% content_for :page_title, @main_question %>
+
+<% content_for :form_card do %>
+  <%= form_with model: @form, url: current_path, local: true, method: "put", builder: VitaMinFormBuilder, html: { class: "form-card form-card--long" } do |f| %>
+    <h1 class="h2"><%= @main_question %></h1>
+
+    <p>
+      <%= t("views.ctc.questions.legal_consent.body.p1") %>
+    </p>
+    <p>
+      <%= t("views.ctc.questions.legal_consent.body.p2") %>
+    </p>
+
+    <div class="form-card__content">
+      <%= f.cfa_input_field(:primary_first_name, t("views.ctc.questions.legal_consent.first_name"), classes: ["form-width--long"]) %>
+      <%= f.cfa_input_field(:primary_middle_initial, t("views.ctc.questions.legal_consent.middle_initial"), classes: ["form-width--short"]) %>
+      <%= f.cfa_input_field(:primary_last_name, t("views.ctc.questions.legal_consent.last_name"), classes: ["form-width--long"]) %>
+      <%= f.vita_min_date_text_fields(
+            :primary_birth_date,
+            t("hub.clients.show.date_of_birth"),
+            ) %>
+      <%= f.cfa_input_field(:primary_ssn, t("views.ctc.questions.legal_consent.ssn"), classes: ["form-width--long"]) %>
+      <%= f.cfa_input_field(:primary_ssn_confirmation, t("views.ctc.questions.legal_consent.ssn_confirmation"), classes: ["form-width--long"]) %>
+      <%= f.cfa_input_field(:sms_phone_number, t("views.ctc.questions.legal_consent.sms_phone_number"), classes: ["form-width--long"]) %>
```

Comment:
based on the form values, this should probably be phone_number instead of sms_phone_number. sms_phone_number is gathered in the CellPhoneNumber form.